index.html

```diff
@@ -4,6 +4,7 @@
   <head>
     <link rel="stylesheet" href="style.css">
     <script src="scripts/pow.js"></script>
+    <script src="sctipts/arr.js"></script>
```

Comment:
🤦 
```suggestion
    <script src="scripts/arr.js"></script>
```

Reply:
I'm burning with shame!! Sorryyy)

Reply:
Because I've already checked to see if I've set the path correctly.